.github/workflows/pr-title-update.yml

```
@@ -0,0 +1,27 @@
+name: Update PR Title If Target Branch Is A Version Branch
```

Maybe a bit shorter and more generic, in case we can add more checks in the future?

```suggestion
name: Update PR title
```